cmd/influx/cli/cli.go

```diff
@@ -198,13 +198,20 @@ func (c *CommandLine) Run() error {
 	c.Version()
 
 	// Only load/write history if HOME environment variable is set.
+	var historyDir string
 	if homeDir := os.Getenv("HOME"); homeDir != "" {
-		// Attempt to load the history file.
-		c.historyFilePath = filepath.Join(homeDir, ".influx_history")
-		if historyFile, err := os.Open(c.historyFilePath); err == nil {
-			c.Line.ReadHistory(historyFile)
-			historyFile.Close()
-		}
+		historyDir = homeDir
+	} else if userDir := os.Getenv("USERPROFILE"); userDir != "" {
```

Comment:
Do you know of an example of another piece of software that reads `USERPROFILE` on Linux because of Windows? I'm not necessarily against this, but I would like to see some precedent for doing this.

I think it might be better to only check `USERPROFILE` if the OS is Windows.